fl4health/feature_alignment/constants.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, Union
+
+from flwr.common.typing import Scalar
+from sklearn.feature_extraction.text import CountVectorizer, HashingVectorizer, TfidfTransformer, TfidfVectorizer
+
+TextFeatureTransformer = Union[CountVectorizer, TfidfTransformer, TfidfVectorizer, HashingVectorizer]
+
+# Constants that specify the data type of tabular data columns.
+NUMERIC = "numeric"
+BINARY = "binary"
+STRING = "string"
+ORDINAL = "ordinal"
+UNKNOWN = "UNKNOWN"
+
+FEATURE_TYPES = [
+    NUMERIC,
+    BINARY,
+    STRING,
+    ORDINAL,
+]
+
+# The default fill-in values for entire missing columns of a certain data type.
+DEFAULT_FILL_VALUES: Dict[str, Scalar] = {BINARY: 0, NUMERIC: 0.0, ORDINAL: UNKNOWN, STRING: "N/A"}
```

Comment:
Below, I see why you defined a DEFAULT_FILL_VALUES dictionary. I think that's fine to do actually, rather than defining the static_method above. It can, however, be a Dict[TabularType, Scalar] instead of just a str->scalar dict.

Reply:
I think this is a really good idea, I'll change the code to using enum instead.